src/cache/precomputed_quantities.jl

```diff
@@ -49,42 +42,23 @@ function implicit_precomputed_quantities(Y, atmos)
         ᶜK = similar(Y.c, FT),
         ᶜts = similar(Y.c, TST),
         ᶜp = similar(Y.c, FT),
-        ᶜh_tot = similar(Y.c, FT),
     )
-    sgs_quantities =
-        turbconv_model isa AbstractEDMF ? (; ᶜtke⁰ = similar(Y.c, FT)) : (;)
-    moisture_sgs_quantities =
-        (
-            turbconv_model isa PrognosticEDMFX &&
-            moisture_model isa NonEquilMoistModel &&
-            microphysics_model isa Microphysics1Moment
-        ) ?
-        (;
-            ᶜq_liq⁰ = similar(Y.c, FT),
-            ᶜq_ice⁰ = similar(Y.c, FT),
-            ᶜq_rai⁰ = similar(Y.c, FT),
-            ᶜq_sno⁰ = similar(Y.c, FT),
-        ) : (;)
+    sgs_quantities = turbconv_model isa AbstractEDMF ? (;) : (;)
```

Comment:
I suppose this is still `sgs_quantities` assignment here since we splat entries later in the code ( `ᶜgradᵥ_θ_virt`, `ᶜgradᵥ_q_tot` etc. ) ; the conditional statement is no longer needed here right ?